ArticleTemplates/assets/js/modules/quiz.js

```diff
@@ -466,8 +465,8 @@ function showScore() {
     initPositionPoller();
 
     // Scroll score panel into view
-    const scroll = new SmoothScroll();
-    scroll.animateScroll('#quiz-scores', null, {speed: 1500, offset: 40});
+    const scoresElement = document.getElementsByClassName('quiz-scores')[0]
+    scoresElement.scrollIntoView()
```

Comment:
It's possible that `scoresElement` will be `undefined` if an element with that class (or `id` if you use `getElementById`) can't be found. If you call a method on `undefined` you'll get a runtime exception (because JavaScript), so you may want to wrap this call:

```suggestion
    if (scoresElement !== undefined) {
      scoresElement.scrollIntoView()
    }
```